autoload/neomake/makers/ft/cpp.vim

```diff
@@ -9,12 +9,14 @@ endfunction
 function! neomake#makers#ft#cpp#clang() abort
     let maker = neomake#makers#ft#c#clang()
     let maker.exe = 'clang++'
+    let maker.args = makers.args + ['-std=c++1z']
     return maker
 endfunction
 
 function! neomake#makers#ft#cpp#gcc() abort
     let maker = neomake#makers#ft#c#gcc()
     let maker.exe = 'g++'
+    let maker.args = makers.args + ['-std=c++1z']
```

Comment:
A test like the following throws:
```
Execute (Error gets reported from maker function):
  function! neomake#makers#ft#neomake_tests#true() abort
      let foo = foo + 'bar'
  endfunction

  set ft=neomake_tests
  RunNeomake true
```

This could be caught somehow and get reported in a better way, but should not happen in the first place, so it is fine.